time_perspective/config.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "name": "ZTPI: Zimbardo Time Perspective Inventory",
+        "template":"survey",
+        "run": [
+                 "https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&amp;lang=en",
+                 "https://fonts.googleapis.com/icon?family=Material+Icons",
+                 "https://code.getmdl.io/1.1.1/material.blue-red.min.css",
+                 "static/css/surveys.css",
+                 "static/js/jquery/ui/smoothness/jquery-ui-1.10.4.custom.min.css",
+                 "static/js/jquery/jquery-2.1.1.min.js",
+                 "https://code.getmdl.io/1.1.1/material.min.js",
+                 "static/js/jquery/ui/jquery-ui-1.10.4.custom.min.js",
+                 "static/js/jquery/jquery.wizard.js",
+                 "static/js/jquery/form/jquery.form-3.50.js",
+                 "static/js/jquery/validate/jquery.validate-1.12.0.min.js"
+               ],
+        "exp_id": "time_perspective",
```

Comment:
since all other surveys have "_survey" appended, we should probably maintain the standard